lib/net.js

```diff
@@ -57,10 +57,10 @@ const normalizedArgsSymbol = internalNet.normalizedArgsSymbol;
 function noop() {}
 
 function createHandle(fd) {
-  var type = TTYWrap.guessHandleType(fd);
+  const type = TTYWrap.guessHandleType(fd);
   if (type === 'PIPE') return new Pipe();
   if (type === 'TCP') return new TCP();
-  throw new TypeError('Unsupported fd type: ' + type);
+  throw new errors.Error('ERR_INVALID_HANDLE_TYPE');
```

Comment:
@pmatzavin look at https://github.com/nodejs/node/blob/cbd3708c85e13b234e9ded0beaaf1780a537dc39/lib/internal/errors.js#L200
You can add just after it:
```js
E('ERR_INVALID_FD_TYPE', 'Unsupported fd type: %s');
```